data/explorations/android_interactions/Test all android interactions.yaml

```diff
@@ -0,0 +1,790 @@
+author_notes: ''
+auto_tts_enabled: true
+blurb: ''
+category: Algorithms
+correctness_feedback_enabled: false
+init_state_name: Introduction
+language_code: en
+objective: test android specific interactions
+param_changes: []
+param_specs: {}
+schema_version: 50
+states:
+  Fraction:
+    card_is_checkpoint: false
+    classifier_model_id: null
+    content:
+      content_id: content
+      html: <p>What fraction represents half of something?</p>
+    interaction:
```

Comment:
Please create a list of all interactions does this file contains, in the PR description.

Reply:
Done, also added a screen recording.